locust/main.py

```diff
@@ -343,15 +363,15 @@ def main():
         console_logger.info(dumps(task_data))
         sys.exit(0)
 
-    if not options.no_web and not options.slave:
+    if not options.no_web and not options.slave and not options.integration:
         # spawn web greenlet
         logger.info("Starting web monitor at %s:%s" % (options.web_host or "*", options.port))
         main_greenlet = gevent.spawn(web.start, locust_classes, options)
     
-    if not options.master and not options.slave:
+    if (not options.master and not options.slave) or options.integration:
```

Comment:
imo, we will have different `LocustRunner` class for integration testing unless we will not have enough data to initialize the`LocustRunner` class if we run it as integration test. I mean integration testing will not require arguments like hatch rate. It will cause an error to the `LocustRunner` class. So, in this part we can make a condition `if options.integration` alone and put the `not options.master and not options.slave` as elif